rfcs/0001-beneficiary-fhir-server-rfc-process-v1.md

```diff
@@ -0,0 +1,168 @@
+
+More seriously, we want to move from the current process where these proposals are "hidden" to one where the entire team can participate. This is expected to:
+
+1. Lead to more thoughtful input on these decisions, and thus better decisions.
+2. Formalize the currently-informal process, leading to clearer outcomes.
+3. A historical record of our thought process and decisions, making it easier for new team members to get up to speed.
+
+## Proposed Solution
+[Proposed Solution]: #proposed-solution
+
+This very-first RFC is a "process only" RFC, in which the team will detail, discuss, and agree on a formal process for making major decisions,
+  e.g. on architecture, process (like this!), and major features.
+Specifically, we're reviewing and hopefully approving these documents:
+
+* [./README.md](Blue Button Data Systems RFCs)
+* [./0000-template.md](Template/Starter RFC)
+* [./0001-beneficiary-fhir-server-rfc-process-v1.md](This RFC: `0001-beneficiary-fhir-server-rfc-process-v1`)
+
+Other future RFCs will likely include:
+
+* "Should we adopt a formal (internal) Code of Conduct / Working Agreement?"
+* "Should we switch from self-managed PostgreSQL-on-EC2 to a more managed DB offering?"
+* "Should we move our Security Impact Assessment (SIA) process to GitHub?"
+* "Should we move our development and/or production systems to Docker?"
+
+Questions to consider when evaluating this particular RFC:
+
+* Is this process inclusive enough?
+    * GitHub's pull requests have a great workflow for engineers, but is it user-friendly enough for non-developers?
+* Knowing our team, will the process work well for the people and personalities involved, or will it somehow turn toxic?
+* How might we change this proposed process to make it more agile?
+* Are we all comfortable having these conversations in public, on GitHub?
+* Does this RFC follow the proposed process well enough to validate the process and associated templates?
+
+### Proposed Solution: Detailed Design
+[Proposed Solution: Detailed Design]: #proposed-solution-detailed-design
+
+Are these two documents correct and sufficient?:
+
+* [./README.md](Blue Button Data Systems RFCs)
+    * Intended to introduce the RFC process at a high level.
+* [./0000-template.md](Template/Starter RFC)
+    * Intended to be copy-pasted for each new proposed RFC.
+
+### Proposed Solution: Compensatory Steps
+[Proposed Solution: Compensatory Steps]: #proposed-solution-compensatory-steps
+
+What do we have to take care in doing, and perhaps do differently than if this were conducted on Confluence?
+
+* No information we wouldn't be comfortable having included as part of a news article
+    (say one by Brian Krebs) should be useed unencrypted --
+    public data needs to be truly public in nature, not just obscurely comfortable.
+* No information that rises to FIPS 199 moderate levels without an internal agreement to reclassify it as public.
```

Comment:
These data disclosure rules should probably be in their own sub-section.

Reply:
Moved to Confluence, per previous comment.